hyppo/conditional/kci.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+from scipy.stats import gamma
+from sklearn.gaussian_process import GaussianProcessClassifier
+from sklearn.gaussian_process.kernels import RBF
+
+from .base import ConditionalIndependenceTest, ConditionalIndependenceTestOutput
+
+
+class KCI(ConditionalIndependenceTest):
+    r"""
+    Kernel Conditional Independence Test Statistic and P-Value.
```

Comment:
This is not descriptive enough, look at the other docstrings for metods in hyppo